### DISCUSSION TITLE
Deployment / usage model

### DISCUSSION THREAD

**Discussion by nathan-williams-rozettatechnology** [lambci/serverless-actions#4] [Q&A] [answered]
Is this something we can configure & run by ourselves, or is there a subscription for the management?
(Not that I am opposed to paying, just working out how to position this for internal discussions)

**Comment by mhart** [comment, marked as answer]
Hey Nathan – the immediate plan is for the server-side aspect (that is, the component that maintains connections with GitHub) to be managed for you – so that it's serverless from your perspective. During the beta this will be free, but eventually it will be paid, with the aim to be an order of magnitude less than it would be to host the same number of servers yourself (ie, aiming for less than $1/mth per runner, hopefully less than $0.50/mth per runner).

Some enterprises have expressed interest in managing that aspect themselves too – so that's an option I'm considering, and it would likely be under an enterprise subscription plan or similar.

**Reply by nathan-williams-rozettatechnology** [reply]
That sounds fantastic, if you have space in your beta, I would be very interested in taking part.
Just to clarify, each runner would cover a single repo?

**Reply by mhart** [reply]
> Just to clarify, each runner would cover a single repo?

Runners can be per organization (in which case, you can use them across repos) or per repo. LambCI will support both.

https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners#about-self-hosted-runners

So you can use them however you like! (Or, more technically, however you set them up 😉)

**Reply by stephenbawks** [reply]
What is going to be the trigger that listens and kicks of LambCI?  Would that be a webhook out to kick off the process (LambCI)?  Assuming LambCI determines the job type and sends it off the correct runner group.  I obviously don't want a long running job that could take 30 minutes going to a Lambda and hence the ability to send it over to Fargate for example.
